substrate/frame/support/src/traits/tokens/fungible/mod.rs

```diff
@@ -58,13 +58,17 @@
 //!   3 holds for 100 units, the account can spend its funds for any reason down to 300 units, at
 //!   which point the holds will start to come into play.
 //!
-//! - **Frozen Balance**: A freeze on a specified amount of an account's free balance until a
-//!   specified block number.
+//! - **Frozen Balance**: A freeze on a specified amount of an account's balance. This amount is 
+//!   set aside and cannot be spent by the account holder.  
```

Comment:
```suggestion
//! - **Frozen Balance**: A freeze on a specified amount of an account's balance. Tokens which 
//!   are frozen cannot be transferred.
```

I think it is best to state things directly. This is I think the most obvious and accurate statement about frozen balances, and does not contradict other ways of manipulating balances like holds.

Reply:
> I think it is best to state things directly. This is I think the most obvious and accurate statement about frozen balances, and does not contradict other ways of manipulating balances like holds.

Yeah! It's clearer that way. I commit it